### PR TITLE
Groceries Mohsen

### PR DESCRIPTION
**Pull request by matefh**
@MohamedBassem

### CODE REVIEW

File: bot/scripts/groceries.coffee [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    cart = robot.brain.get('cart')
+    data = JSON.stringify({
+      "Address": null,
+      "SelectedAddress": 4343,
+      "SelectedPhone": 5447,
+      "TimeSlotID": 45,
+      "PaymentMethod": "CashOnDelivery",
+      "WarehouseId": 1,
+      "EnterpriseDiscount": null,
+      "Items": cart.map (i) -> {"Id": i["id"], "Qty": i["count"], "Coupon": null}
```
Comment (matefh) [inline bot/scripts/groceries.coffee]: We need to add the user id and use it here

File: bot/scripts/groceries.coffee [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+#   These are from the scripting documentation: https://github.com/github/hubot/blob/master/docs/scripting.md
+
+
+
+module.exports = (robot) ->
+  robot.respond /(pls|please|plz) order (.*)/i, (res) ->
+    robot.brain.set('milk', {id: '26007', count: '4'})
+    robot.brain.set('cheddar cheese', {id: '25713', count: '1'})
+    robot.brain.set('mix strawberry', {id: '19555', count: '4'})
+    robot.brain.set('milk toast', {id: '17120', count: '1'})
```
Comment (MohamedBassem) [inline bot/scripts/groceries.coffee]: Let's change those 4 lines to : 
```
robot.brain.set('stock', {
  cheese: {...},
  milk: {...} 
})
```

**Comment by matefh** [comment]
@MohamedBassem I did some improvements. I think we can close #4 #7 #8

**Comment by MohamedBassem** [comment]
LGTM ( kan nfsy 22olha awi :D ) :+1:

**Comment by matefh** [comment]
:D

**Pull request merged**